code_schemes/gender.json

```diff
@@ -18,9 +18,9 @@
     {
       "CodeID": "code-01667903",
       "CodeType": "Normal",
-      "DisplayText": "female",
+      "DisplayText": "woman",
       "NumericValue": 2,
-      "StringValue" : "female",
+      "StringValue" : "woman",
       "Shortcut": "f",
```

Comment:
Should this be changed to w?

I think yes but check with RDA if unsure. I don't think shortcuts are widely used anyway

Reply:
I think 'w' too. Updated thanks.